torch/csrc/autograd/init.cpp

```diff
@@ -49,6 +50,10 @@ PyObject * THPAutograd_initExtension(PyObject *_unused)
   });
   m.def("_pop_range", []() { torch::autograd::profiler::popRange(); });
 
+  m.def("_unsafe_set_delete_function_max_recursion_depth", [](size_t value) {
+    deleteFunctionMaxRecursionDepth = value;
```

Comment:
Maybe add a big TODO comment about replacing this with a remaining-stack-size detector algorithm